hiera.yaml

```diff
@@ -14,6 +14,8 @@ hierarchy:
     path: "terraform_data.yaml"
   - name: "OS version"
     path: "os/%{facts.os.name}/%{facts.os.release.major}.yaml"
+  - name: "Software stack"
```

Comment:
Because the `profile::cvmfs::client::lmod_default_modules` of the software stack should have precedence on the `profile::cvmfs::client::lmod_default_modules`, the software stack yaml files should be higher up in the hierarchy than the os files.